src/daemon/keymap.c

```diff
@@ -453,7 +453,7 @@ const key keymap_bragi[N_KEYS_BRAGI_PATCH] = {
     { "prev",          126, KEY_PREVIOUSSONG },
     { "mr",             -1, KEY_CORSAIR },
     { "profswitch",    128, KEY_CORSAIR },
-    { 0,                -1, KEY_NONE },
+    { "knobpress",     129, KEY_MUTE },
```

Comment:
This should either be "mute" (by using the keymap_patch mechanism) or knobpress with KEY_CORSAIR.

Reply:
Note: If we set it to KEY_CORSAIR then it won't perform the default action of toggling mute in daemon mode, but I could patch like you said.